translate/storage/html.py

```diff
@@ -313,6 +314,7 @@ def startblock(self, tag, attrs=None):
         self.currentcomment = ""
         self.currenttag = tag
         self.currentpos = self.getpos()[0]
+        self.currentoffset = self.getpos()[1]
```

Comment:
`self.currentpos, self.currentoffset = self.getpose()[:1]` might be neater, up to you.